lib/node_modules/@stdlib/math/base/special/trunc/manifest.json

```diff
@@ -1,7 +1,6 @@
 {
   "options": {
-    "task": "build",
-    "wasm": false
```

Comment:
@gunjjoshi Don't delete the wasm fields or configuration. These were added intentionally in preparation for future WebAssembly builds.

Reply:
Oh, sorry for that, I've reverted the changes, thanks!